### ISSUE TITLE
Vcf files with "chr" in CHROM already

### ISSUE THREAD

**Issue by pwwang** [hammerlab/vcf-annotate-polyphen#4]
https://github.com/hammerlab/vcf-annotate-polyphen/blob/efcf086ec537075d4677f539af4c1c4dcdfbcdad/vap/cli.py#L52

Can we patch this to the above line

```python
'chr{}'.format(v.CHROM.replace('chr', '')), 
```
or 
```python
'chr{}'.format(v.CHROM) if not v.CHROM.startswith('chr') else v.CHROM, 
```

to adopt vcf files with `chr` in CHROM already?

**Comment by armish** [comment]
sure @pwwang -- feel free to send a PR my way (the second solution would be much better). Sorry that I wasn't handling this case as this annotator was only tested on one type of VCF. 

Having said that, this package/annotation is really old. In case you want to annotate your VCFs, I would recommend going with a more recent implementation, such as https://github.com/brentp/vcfanno

**Comment by armish** [comment]
closed via #5
